back/courses/templates/courses/view_course.html

```diff
@@ -0,0 +1,103 @@
+{% load static %}
+<!--Markdownify-->
+{% load markdownify %}
+<!--Header-->
+{% if course.acronym %}{% include "header.html" with title=course.acronym %}{% else %}{% include "header.html" with title=course.name %}{% endif %}
```

Comment:
Perhaps, when the acronym is given, the title should be "acronym - name" instead of just "acronym"

Reply:
Yes !